QEfficient/utils/_utils.py

```diff
@@ -188,3 +188,40 @@ def padding_check_and_fix(tokenizer: Union[PreTrainedTokenizer, PreTrainedTokeni
             tokenizer.pad_token_id = tokenizer.eos_token_id
         else:
             tokenizer.pad_token_id = tokenizer.vocab_size - 1
+
+
+def get_config(config):
```

Comment:
change this function to return `padding_shape`, and rename it as `get_padding_shape_from_config`

Reply:
this function returns number of layers along with other parameters needed for padding shape.

Reply:
Move this call inside InputHandler initializer as discussed.